<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>601</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="open_btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rec_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rec On</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="capture_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
